### PR TITLE
tensor shape was null for rsa modulus

### PR DESCRIPTION
**Pull request by rdragos**
Closes  capeprivacy/planning#1394 and #71

### CODE REVIEW

File: tf_big/cc/big_tensor.h [hunk 1]
```diff
@@ -48,6 +51,27 @@ typedef Matrix<mpz_class, Dynamic, Dynamic> MatrixXm;
...
+  }
+}
+inline void init_randstate(gmp_randstate_t& state) {
+  gmp_randinit_mt(state);
+  unsigned long seed;
+  reseed(seed);
+  gmp_randseed_ui(state, seed);
+}
+}  // namespace gmp_utils
+
```
Comment (jvmncs) [inline tf_big/cc/big_tensor.h:74]: is this related to capeprivacy/planning#1394? maybe the name/description for the PR should be updated in that case?

**Comment by rdragos** [comment]
Closes #1394 and #71
